### PR TITLE
Sublevel card bug / feedback style fixes

### PR DESCRIPTION
**Pull request by maureensturgeon**
- Updates sublevel summarize to provide more accurate status (reflected on sublevel card see below)
Screen Shot 2021-08-13 at 1 13 15 PM

- Updates style of "seen by student" to center vertically next to button
Screen Shot 2021-08-13 at 1 17 10 PM

## Links

- jira ticket: [LP-1998](https://codedotorg.atlassian.net/browse/LP-1998)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/models/levels/bubble_choice.rb [hunk 1]
```diff
@@ -143,16 +143,13 @@ def summarize_sublevels(script_level: nil, user_id: nil, should_localize: false)
...
           script: script_level.try(:script),
           user_id: user_id
-          )&.perfect?
-        level_info[:status] = if level_info[:perfect]
-                                SharedConstants::LEVEL_STATUS.perfect
-                              else
-                                SharedConstants::LEVEL_STATUS.not_tried
-                              end
+          )
+        level_info[:perfect] = user_level&.perfect?
```
Comment (jamescodeorg) [inline dashboard/app/models/levels/bubble_choice.rb:151]: Do we still need `level_info[:perfect]`?

**Review by jamescodeorg — APPROVED** [review]
Looks great!

File: dashboard/app/models/levels/bubble_choice.rb (hunk 1 above)
Comment (maureensturgeon) [reply dashboard/app/models/levels/bubble_choice.rb:151]: I saw it's being used here https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/SublevelCard.jsx#L90 and I didn't bother doing a refactor this time

Comment (maureensturgeon) [reply dashboard/app/models/levels/bubble_choice.rb:151]: I figured we'd be revisiting lesson extras soon anyway

**Pull request merged**